tox.ini

```diff
@@ -89,3 +89,13 @@ commands =
         --confcutdir=tests/integration \
         {posargs:tests/integration}
 passenv = KRB5CCNAME REQUESTS_CA_BUNDLE KRB5_CLIENT_KTNAME CACHITO_TEST_CERT CACHITO_TEST_KEY JOB_NAME
+
+[pytest]
+addopts = --tb=native --verbose --capture=no
+testpaths = tests
+log_cli = True
+log_level = NOTSET
+log_format = %(asctime)s %(levelname)s %(message)s
+log_date_format = %Y-%m-%d %H:%M:%S
+env =
+  D:CACHI2_IMAGE=quay.io/containerbuildsystem/cachi2:latest
```

Comment:
D:

( ignore this, I just found the D: smiley funny :D )

Reply:
lool :D